src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/services/UploadEnvelopeDocumentsServiceTest.java

```diff
@@ -245,7 +227,16 @@ void should_mark_as_uploaded_when_everything_went_well() throws Exception {
 
         // and
         verify(documentProcessor, times(1)).uploadPdfFiles(emptyList(), emptyList());
-        verify(blobManager).tryReleaseLease(blockBlob, CONTAINER_1, ZIP_FILE_NAME, LEASE_ID);
+
+    }
+
+    void leaseAcquired() {
```

Comment:
you are mocking here right ? leaseAcquired doesn't sound right to me. may be `mockAcquiringLease`  or mockLeaseInvocation or something ?